src/marqo/tensor_search/api.py

```diff
@@ -185,6 +185,10 @@ def add_or_replace_documents(
         ),
         mappings: typing.Optional[dict] = Depends(api_utils.decode_mappings)):
     """add_documents endpoint (replace existing docs with the same id)"""
+    if processes is not None:
```

Comment:
This needs a test

Reply:
We just delete this error and make it easier.